presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Fix history leak in coordinator for failed or canceled queries.
+* Fix memory leak related to query tracking in coordinator that was introduced
+  in 0.213.
+* Fix planning failures when lambdas are used in join filter expression.
+* Send proper error response to client for certain types of errors during query
+  creation.
+* Improve error message when comparator provided to :func:`array_sort` function
```

Comment:
Improve error message when invalid comparator is provided to the :func:`array_sort` function.